infrastructure/builder/src/build/monoimage.js

```diff
@@ -150,6 +150,9 @@ const generateMonoimageTasks = ({tasks, baseDir, spec, cfg, cmdOptions}) => {
     },
   });
 
+  // TODO: add hooks push for web-ui
+  // TODO: add entry point for docker run for web-ui
+
```

Comment:
It might be better to record these in a bugzilla bug or github issue than in the source..

Reply:
I wanted to mark the specific place in the code where I need to look while I still remember it 😔